Essentials/src/messages.properties

```diff
@@ -553,7 +553,7 @@ userdataMoveBackError=Failed to move userdata/{0}.tmp to userdata/{1}\!
 userdataMoveError=Failed to move userdata/{0} to userdata/{1}.tmp\!
 userDoesNotExist=\u00a74The user\u00a7c {0} \u00a74does not exist.
 userIsAway=\u00a77* {0} \u00a77is now AFK.
-userIsAwayWithMessage=\u00a77* {0} \u00a77is now AFK.
+userIsAwayWithMessage=\u00a77* {0} \u00a77is now AFK, reason {1}
```

Comment:
I'd prefer `... is now AFK. Reason: {1}`, but I guess it's a matter of preference.